tokio-util/src/udp/frame.rs

```diff
@@ -48,7 +51,11 @@ pub struct UdpFramed<C> {
 const INITIAL_RD_CAPACITY: usize = 64 * 1024;
 const INITIAL_WR_CAPACITY: usize = 8 * 1024;
 
-impl<C: Decoder + Unpin> Stream for UdpFramed<C> {
+impl<C, T> Stream for UdpFramed<C, T>
+where
+    T: Borrow<UdpSocket> + Unpin,
```

Comment:
You do not need the `Unpin` bound.
```suggestion
    T: Borrow<UdpSocket>,
```
You can even add an unconditional `impl<C, T> Unpin for UdpFramed<C, T> {}`

Reply:
That last impl is what I was missing, without it the `self.get_mut()` line in `Stream` complains about `Unpin`. Thanks!